applications/sdp/gpio/README.rst

```diff
@@ -0,0 +1,25 @@
+.. _sdp_gpio:
```

Comment:
file must be included in the index ``nrf\doc\nrf\applications.rst``, as follows:
``../../applications/sdp/gpio/README``